README.md

```diff
@@ -68,6 +68,7 @@ class ExampleServiceClient
       error_threshold:  50,
 
       # seconds before the circuit times out
+      # if set to nil no timeout is used
```

Comment:
It feels wrong to me for a configuration option named ```timeout_seconds``` to accept anything but Integer (fixnum/bignum on <2.4), Float, or nil values. I agree that using ```:disabled``` is more declarative though.

I would rather the name be ```timeout``` or some maybe a variant of ```*_window``` rather then ```timeout_seconds``` as those make more sense allowing more types of values. The definition would change to something like:
``` 
Seconds before the circuit times out (using Ruby Timeout).
Setting to :disabled disables this functionality in the circuit
```